Packet Downloader running out of space

We are getting several 100MB packet files occasionally, and if multiple instruments are downloaded with large packet volumes, we will saturate the ephemeral storage on our Lambda.

We should increase the size of the ephemeral storage to try and minimize this issue.
https://github.com/IMAP-Science-Operations-Center/sds-data-manager/blob/f70ef014eb744ddbff3bf8a0694a5f9a48afdfec/sds_data_manager/constructs/packet_downloader_lambda_construct.py#L55


```
2026-01-05T18:11:46.422Z | [INFO] 2026-01-05T18:11:46.422Z 94a6de84-0870-47db-8e5e-cf3bc8818c4f Saving binary data of size 2620 kB to /tmp/imap/glows/l0/2026/01/imap_glows_l0_raw_20260102-repoint00097_v001.pkts
  | 2026-01-05T18:11:46.426Z | [ERROR] OSError: [Errno 28] No space left on device Traceback (most recent call last):   File "/var/task/SDSCode/pipeline_lambdas/packet_downloader.py", line 199, in lambda_handler     download_ena_data(event_repoint_key)   File "/var/task/SDSCode/pipeline_lambdas/packet_downloader.py", line 146, in download_ena_data     webpoda.download_repointing_data(   File "/opt/python/imap_data_access/webpoda.py", line 492, in download_repointing_data     path.write_bytes(pointing_packet_content)   File "/var/lang/lib/python3.12/pathlib.py", line 1037, in write_bytes     return f.write(view)
  |  
```

--- a/sds_data_manager/constructs/packet_downloader_lambda_construct.py
+++ b/sds_data_manager/constructs/packet_downloader_lambda_construct.py
@@ -68,6 +68,9 @@ class PacketDownloaderLambda(Construct):
             # We are downloading data, so make sure we have a longer timeout here
             timeout=cdk.Duration.minutes(15),
             memory_size=2048,
+            # Some packet files can be large depending on how long it has been since
+            # the last downlink. Increase the ephemeral /tmp size to accomodate this.
+            ephemeral_storage_size=cdk.Size.gibibytes(2),
             # Environment variables - API key will be retrieved from SSM at runtime
             environment={
                 "IMAP_DATA_ACCESS_URL": data_access_url,

